src/layouts/Base.astro

```diff
@@ -55,69 +57,68 @@ const pathname = Astro.url.pathname.replace(/\/$/, "")
 
 	<body>
 		<Header />
+		<NoiseContainer>
```

Comment:
not really a big deal given the size of this project, but I wonder if there's a way we could avoid all the extra `<div>`s here

Could a component like `<Container noise grid>` wrap it up into a single `<div class="relative">` wrapper?

Reply:
Not ideal, the grid is only meant to be applied to the hero section, and the noise applies to the entire thing

Reply:
Ah yep, forgot the grid is only part of the viewport!